src/software/SfM/main_ComputeStructureFromKnownPoses.cpp

```diff
@@ -23,11 +23,13 @@ int main(int argc, char **argv)
   CmdLine cmd;
 
   std::string sSfM_Data_Filename;
+  std::string sFeaturesDir;
   std::string sMatchesDir;
   std::string sMatchesGeometricModel = "f";
   std::string sOutFile = "";
 
   cmd.add( make_option('i', sSfM_Data_Filename, "input_file") );
+  cmd.add( make_option('f', sFeaturesDir, "feat_dir") );
```

Comment:
Here, it's important to use 2 different arguments, because the matches are optional but not the folder with features and descriptors.